src/crimson/osd/pg_backend.cc

```diff
@@ -119,18 +119,22 @@ PGBackend::mutate_object(
   std::set<pg_shard_t> pg_shards,
   crimson::osd::ObjectContextRef &&obc,
   ceph::os::Transaction&& txn,
-  const MOSDOp& m,
+  OSDOpParams&& osd_op_p,
   epoch_t min_epoch,
   epoch_t map_epoch,
-  eversion_t ver)
+  vector<pg_log_entry_t> logv)
 {
+  auto& m = osd_op_p.req;
   logger().trace("mutate_object: num_ops={}", txn.get_num_ops());
   if (obc->obs.exists) {
 #if 0
     obc->obs.oi.version = ctx->at_version;
     obc->obs.oi.prior_version = ctx->obs->oi.version;
 #endif
 
+    obc->obs.oi.prior_version = obc->obs.oi.version;
+    obc->obs.oi.version = osd_op_p.at_version;
+
```

Comment:
might want to minimize the vertical white spaces. see https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace